types/expo/index.d.ts

```diff
@@ -1855,19 +1856,36 @@ export namespace Notifications {
         };
     }
 
+    interface SchedulingOptions {
+        time: Date | number;
+        repeat?: 'minute' | 'hour' | 'day' | 'week' | 'month' | 'year';
+        intervalMs?: number;
+    }
+
+    interface ChannelAndroid {
+        name: string;
+        description?: string;
+        sound?: boolean;
+        priority?: 'min' | 'low' | 'default' | 'high' | 'max';
+        vibrate?: boolean | number[];
+        badge?: boolean;
+    }
+
     type LocalNotificationId = string | number;
 
     function addListener(listener: (notification: Notification) => any): EventSubscription;
     function getExpoPushTokenAsync(): Promise<string>;
     function presentLocalNotificationAsync(localNotification: LocalNotification): Promise<LocalNotificationId>;
     function scheduleLocalNotificationAsync(
         localNotification: LocalNotification,
-        schedulingOptions: { time: Date | number, repeat?: 'minute' | 'hour' | 'day' | 'week' | 'month' | 'year' }
+        schedulingOptions: SchedulingOptions
     ): Promise<LocalNotificationId>;
     function dismissNotificationAsync(localNotificationId: LocalNotificationId): Promise<void>;
     function dismissAllNotificationsAsync(): Promise<void>;
     function cancelScheduledNotificationAsync(localNotificationId: LocalNotificationId): Promise<void>;
     function cancelAllScheduledNotificationsAsync(): Promise<void>;
+    function createChannelAndroidAsync(id: string, channel: ChannelAndroid): Promise<void>;
```

Comment:
It seems to me that there isn't really any natural way to sort these functions. How about sorting them alphabetically?

Reply:
It's in the order from expo docs. I can sort them any other way.

Reply:
Expo docs it a great order. This way, it will easier for a user to orientate in both of docs and types.

Reply:
I hadn't caught that, @burtek. And I agree that keeping the same order as the Expo documentation is better than sorting alphabetically.